docs: updated network diagram (#1047)
spectrocloud/librarium: content/docs/04-clusters/06-cluster-management/11-pack-monitoring.md

# Pack Monitoring

> Palette provides a well defined color scheme to monitor the deployment status of packs during Palette Workload Cluster deployment. Different colors represent the different stages of pack installation status. It clearly defines the progress of installation. As the number of packs added to a cluster via cluster profile increases, this monitoring technique is highly user friendly for status recognition.

Reviewer: ```suggestion
Palette provides a color scheme to help you monitor pack installation progress during Palette Workload Cluster deployment. Different colors represent stages of pack installation so you can track the progress of packs as they are added to a cluster.
```
